ui-tests/src/test/java/org/openmrs/reference/page/EditPatientRelationshipPage.java

```diff
@@ -0,0 +1,34 @@
+package org.openmrs.reference.page;
+
+import org.openmrs.uitestframework.page.Page;
+import org.openqa.selenium.By;
+import org.openqa.selenium.support.ui.Select;
+
+public class EditPatientRelationshipPage extends Page {
+	
+    private  HomePage homePage;
+    private static String name = "John";
+    private  static  final By SELECT_RELATIONSHIP_TYPE = By.id("relationship_type");
+    private static final By PERSON_NAME = By.cssSelector("#relationship > p:nth-child(2) > input.person-typeahead.ng-pristine.ng-valid.ng-empty.ng-touched");
+    private static final By NEXT_BUTTON = By.id("next-button");
+    private static final By CONFIRM_BUTTON = By.id("registration-submit");   
+
+    public EditPatientRelationshipPage(Page parent) {
+        super(parent);
+    }
+
+    public void clickOnSelectRelationshipType() throws InterruptedException{
+        clickOn(SELECT_RELATIONSHIP_TYPE);
+        Select relationshipType = new Select(driver.findElement(By.id("relationship_type")));
+        relationshipType.selectByVisibleText("Doctor");
+        setTextToFieldNoEnter(PERSON_NAME, name);
+        clickOn(NEXT_BUTTON);
+        clickOn(CONFIRM_BUTTON);
+    }
+    
+    @Override
+    public String getPageUrl() {
+        return "/registrationapp/editSection.page";
+    }
+}
+
```

Comment:
If you can remove the white spaces within the last empty code line 34 will be sweet and the line will obey the principle of silence and will not be seen public show but rather it will hide under covers

Reply:
Eh you can try editting this line and see if we have morethan one line.

Reply:
Its actually one but git is showing it public because it contains white spaces which pro-grammatically should be removed to make the line completely empty. If its made empty it will be hidden from the public seen But this is not a big deal but some one who is interested with grasping more skills with git would invent in whatever can be grabbed. Personally I learnt this from @ibacher

Reply:
This being  a minor comment may not need a change but some geeks prefer doing things in a smart way and if I were @sherrif10  would be open to learn how its done.

Reply:
Hello @kdaud , we can make it more clear as suggested earlier by @ibacher

Reply:
The magic is just removing white spaces within the line and the line will become silent